### ISSUE TITLE
Why is the training loss nan

### ISSUE THREAD

**Issue by qmechlab** [lionel-w2/FAP#2]

**Comment by xinwong** [comment]
I'm encountering the same issue while training on the ImageNet dataset.

**Comment by xinwong** [comment]
I resolved the NaN issue by downgrading torch from version 2.1 to 1.13.

**Comment by qmechlab** [comment]
Are you only experiencing this only with imagenet? I'm experiencing this on all datasets except dtd. It's strange. I downgraded from pytorch 2.5 to 1.10 and haven't had a problem yet.

**Comment by xinwong** [comment]
> Are you only experiencing this only with imagenet? I'm experiencing this on all datasets except dtd. It's strange. I downgraded from pytorch 2.5 to 1.10 and haven't had a problem yet.

I have primarily experimented with ImageNet, and downgrading torch to version 1.13 successfully resolved the issue.

**Comment by xiaoyaoyou9** [comment]
maybe it is up to the precision. When I train it on torch2.10 with fp16 on H20 will get NaN. But I  use the fp32 will not get nan
